src/app/src/components/Sidebar.js

```diff
@@ -21,7 +22,17 @@ const Sidebar = ({ fish }) => {
     const date = moment(timestamp)
         .format('MMMM Do')
         .toUpperCase();
-    const videoPlayer = fish && <VideoPlayer src={video} />;
+    const videoPlayer =
+        fish && commonName === LIVE_FEED_LABEL ? (
```

Comment:
Can you add a comment here describing what we are checking for here and why in some cases we load the video as an image and in other cases a video?